### PR TITLE
microverse-portfolio

### PR DESCRIPTION
**Pull request by Joaquin123455**
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
    - Linter should be set up correctly for this repository.
    - Linter result should be green.
    - There should be no custom changes to linter config.
- Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
    - README file should be descriptive.
    - Commit messages should be meaningful.
    - Pull Request should have a descriptive title explaining what changes are inside.
    - Pull Request should have a short summary describing introduced changes in more detail.
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
    - Branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
    - Pull Request is made from 'feature branch' into the `main` branch.

### HTML/CSS requirements

- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <link rel="stylesheet" href="style.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1 class="header-title">My Logo</h1>
+    <i class="fa fa-bars fa-lg bars"></i>
+  </header>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! The `hamburger` button should have a `nav` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices 👍🏼

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <main>
+    <div class="main-page">
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! The container of the section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. The container of the section should be a `section` tag 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Joaquin123455, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Descriptive `PR` summary ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1 class="header-title">My Logo</h1>
```
Comment (Qoosim) [inline index.html]: - [OPTIONAL] Wonderful job implementing requests by the previous reviewers. This particular requirement escaped their views and as a result the next reviewer must make it optional in order to maintain consistency in our review. However, for conventional, semantics and better user experience reasons, it is required for any logo on a web page to link to the homepage by adding anchor tag (<a>). Kindly add anchor tag to your logo on the page. Therefore, I would recommend you to attend to it and fix.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Joaquin123455, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**